src/GovUk.Education.ExploreEducationStatistics.Data.Services/Mappings/DataServiceMappingProfiles.cs

```diff
@@ -33,12 +33,12 @@ public DataServiceMappingProfiles()
                     }
                 });
 
-            CreateMap<Publication, TopicPublicationViewModel>();
+            CreateMap<Content.Model.Publication, TopicPublicationViewModel>();
 
-            CreateMap<Theme, ThemeViewModel>()
+            CreateMap<Content.Model.Theme, ThemeViewModel>()
                 .ForMember(dest => dest.Topics, opts => opts.Ignore());
 
-            CreateMap<Topic, TopicViewModel>()
+            CreateMap<Content.Model.Topic, TopicViewModel>()
```

Comment:
I don't think this ThemeService should be in Data Services at all now since no themes in the Stats database.
ThemeController already exists in the Content API so we should switch over to using that (which I see already has a method to GetThemes but might need to see what the differences are)